cmd/query/app/static_handler.go

```diff
@@ -28,7 +28,7 @@ import (
 var (
 	// The following patterns are searched and replaced in the index.html as a way of customizing the UI.
 	configPattern      = regexp.MustCompile("JAEGER_CONFIG *= *DEFAULT_CONFIG;")
-	configJsPattern    = regexp.MustCompile(`(?im)^\s*\/\/\s*JAEGER_CONFIG_JS.*\n.*`)
+	configJsPattern    = regexp.MustCompile(`(?im)^\s*//\s*JAEGER_CONFIG_JS.*\n.*`)
```

Comment:
please add a test from #7435 to ensure this doesn't break search & replace capabilities